kivy/graphics/cgl_backend/cgl_angle.pyx

```diff
@@ -0,0 +1,153 @@
+"""
+CGL/ANGLE: GL backend implementation using ANGLE.
+"""
+
+include "../common.pxi"
+include "../../include/config.pxi"
+
+from kivy.graphics.cgl cimport *
+
+cdef extern from *:
+    """
+    #if __USE_ANGLE_GL_BACKEND
+        #include "EGL/egl.h"
+    #endif
+    """
+    void *eglGetProcAddress(const char * procname)
+
+cpdef is_backend_supported():
+    return USE_ANGLE_GL_BACKEND
+
+def init_backend():
+    if not USE_ANGLE_GL_BACKEND:
+        raise Exception('ANGLE GL backend is not supported')
```

Comment:
This condition is confusing me :thinking: 
Am I reading it the wrong way?
I feel like there's a mismatch between the flag name and the exception message given that we're checking that the flag is not set. Is it because that condition would actually return False if the flag is set to anything including 0?

Reply:
This raises an exception when the developer tries to use the `angle` provider even if `USE_ANGLE_GL_BACKEND!=True`.

We do something similar here:
https://github.com/kivy/kivy/blob/17319904d26f257c1a26b94689ca9f3f6f3f96ff/kivy/graphics/cgl_backend/cgl_glew.pyx#L26

Reply:
All right now I get it thanks!